scripts/ci/setup_build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+set -ex
+
+pip install -r requirements.txt
```

Comment:
if there is py code should you be running py linter too ?

Reply:
The only Python code is Sphinx and its extensions (the docs engine).

Reply:
Are we modifying these files directly?

https://github.com/symbol/symbol-docs/blob/3c8e3acf21c67eec066e3c35f36a02e6e4f33a72/source/_ext/card.py#L3

Reply:
Yeah, that's a Sphinx extension I wrote. Feel free to lint anything in the `_ext` folder but, in theory, those files should rarely be modified.

Reply:
Started fixing python linting errors.  The more I fix the more shows up :(